### PR TITLE
Created component ToggleButton, component which show latitude and lon…

### PR DESCRIPTION
**Pull request by artemmatvieiev**
…gitude, component which show all posts of selected user

### CODE REVIEW

**Review by iliatcymbal — APPROVED** [review]
Please, fix all comments and merge

File: src/ToggleButton/ToggleButton.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+  }
+
+  clickHandler = () => {
+    const { className, text, toggleText } = this.state;
+    const { toggleComponentId } = this.props;
+
+    const elem = document.getElementById(toggleComponentId);
+
+    if (/active/i.test(className)) {
```
Comment (iliatcymbal) [inline src/ToggleButton/ToggleButton.js]: do not use reg exp for such simple validation (instead you can use includes method)

File: src/List/List.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export const List = ({ items, clickHandler, className, field = 'name' }) => (
+  <ul className={ className || null }>{ items.map( item => 
+    <li 
+      key={ item.id || Date.now() + Math.random() }
```
Comment (iliatcymbal) [inline src/List/List.js]: it is not good idea to use random in this place (in render method as well). If you don't have id it's better to use index

File: src/ToggleButton/ToggleButton.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      text: props.text || 'Button',
+      toggleText: props.toggleText || props.text || 'Button'
+    }
+  }
+
+  clickHandler = () => {
+    const { className, text, toggleText } = this.state;
+    const { toggleComponentId } = this.props;
+
+    const elem = document.getElementById(toggleComponentId);
```
Comment (iliatcymbal) [inline src/ToggleButton/ToggleButton.js]: you don't need to perform DOM operations with react

File: src/ToggleButton/ToggleButton.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        text: this.props.text || 'Button'
+      });
+      toggleComponentId ? elem.classList.remove('active') : null
+    }
+    else {
+      this.setState({
+        className: `${ className } active`,
+        text: `${ toggleText || text }`
+      });
+      toggleComponentId ? elem.classList.add('active') : null
```
Comment (iliatcymbal) [inline src/ToggleButton/ToggleButton.js]: don't use native DOM elements in react

File: src/main/Main.js [hunk 5]
```diff
@@ -3,17 +3,97 @@ import React, { Component } from 'react';
...
+    this.getGeolocation();
+  }
+
+  getUsers = () => {
+    this.setState({ 
+      loadingUsers: true,
+      users: [] 
+    });
+
+    fetch('https://jsonplaceholder.typicode.com/users')
```
Comment (iliatcymbal) [inline src/main/Main.js:33]: you should make request for specific user `/posts?userId=1`

**Review by artemmatvieiev — COMMENTED** [review]
Fixed all comments

**Pull request merged**